joss/paper.md

```diff
@@ -1,51 +1,117 @@
 ---
-title: '`Eureka!`: An Exoplanet Transit Spectroscopy Pipeline'
+title: '`Eureka!`: An End-to-End Pipeline for JWST Time-Series Observations'
 tags:
+  - Python
   - JWST
-  - python
+  - HST
   - astronomy
   - exoplanets
   - spectroscopy
+  - photometry
 authors:
-
-  - name: Eureka Collaboration 
+  - name: Taylor J. Bell
+    orcid: 0000-0003-4177-2149
+    affiliation: 1
+  - name: Eva-Maria Ahrer
+    orcid: 0000-0003-0973-8426
+    affiliation: 2
+  - name: Jonathan Brande
+    orcid: 0000-0002-2072-6541
+    affiliation: 3
+  - name: Aarynn L. Carter
+    orcid: 0000-0001-5365-4815
+    affiliation: 4
+  - name: Adina D. Feinstein
+    orcid: 0000-0002-9464-8101
+    affiliation: 5
+  - name: Giannina Guzman
+    orcid: 0000-0001-6340-8220
+    affiliation: 6
+  - name: Megan Mansfield
+    orcid: 0000-0003-4241-7413
+    affiliation: 7
+  - name: Sebastian Zieba
+    orcid: 0000-0003-0562-6750
+    affiliation: 8
+  - name: Caroline Piaulet
+    orcid: 0000-0002-2875-917X
+    affiliation: 9
+  - name: Joseph Filippazzo
+    orcid: 0000-0002-0201-8306
+    affiliation: 10
+  - name: Erin M. May
+    orcid: 0000-0002-2739-1465
+    affiliation: 11
+  - name: Kevin B. Stevenson
```

Comment:
If we're going by alphabetical order, I should be after Laura in the final group.